packages/ckeditor5-html-support/src/schemadefinitions.js

```diff
@@ -0,0 +1,812 @@
+ */
+
+/**
+ * @module html-support/schemadefinitions
+ */
+
+// Skipped elements due to HTML deprecation:
+// * noframes (not sure if we should provide support for this element. CKE4 is not supporting frameset and frame,
+//   but it will unpack <frameset><noframes>foobar</noframes></frameset> to <noframes>foobar</noframes>, so there
+//   may be some content loss. Although using noframes as a standalone element seems invalid)
+// * keygen (this one is also empty)
+// * applet (support is limited mostly to old IE)
+// * basefont (this one is also empty)
+// * isindex (basically no support for modern browsers at all)
+//
+// Skipped elements due to lack empty element support:
+// * hr
+// * area
+// * br
+// * command
+// * map
+// * wbr
+// * colgroup -> col
+//
+// Skipped elements due to complexity:
+// * datalist with option elements used as a data source for input[list] element
+//
+// Skipped elements as they are handled as an object content:
+// * track
+// * source
+// * option
+// * param
+// * optgroup
+//
+// Skipped full page HTML elements:
+// * body
+// * html
+// * title
+// * head
+// * meta
+// * link
+// * etc...
+//
+// Skipped hidden elements:
+// noscript
+// script
+
+export default {
+	block: [
+		// Existing features
+		{
+			model: 'heading1',
+			view: 'h2'
+		},
+		{
+			model: 'heading2',
+			view: 'h3'
+		},
+		{
+			model: 'heading3',
+			view: 'h4'
+		},
+		{
+			model: 'codeBlock',
+			view: 'pre'
+		},
+		{
+			model: 'paragraph',
+			view: 'p'
+		},
+		{
+			model: 'blockQuote',
+			view: 'blockquote'
+		},
+		{
+			model: 'listItem',
+			view: 'li'
+		},
+		{
+			model: 'pageBreak',
+			view: 'div'
+		},
+		{
+			model: 'rawHtml',
+			view: 'div'
+		},
+
+		// Compatibility features
+		{
+			model: '$htmlSection',
+			modelSchema: {
+				allowChildren: '$block',
+				allowIn: [ '$root', '$htmlSection' ],
+				isBlock: true
+			}
+		},
+		{
+			model: 'htmlP',
+			view: 'p',
+			modelSchema: {
+				inheritAllFrom: '$block'
+			}
+		},
+		{
+			model: 'htmlBlockquote',
+			view: 'blockquote',
+			modelSchema: {
+				inheritAllFrom: '$htmlSection'
+			}
+		},
+		{
+			model: 'htmlTable',
+			view: 'table',
+			modelSchema: {
+				allowIn: [ '$htmlSection', '$root' ],
+				isBlock: true
+			}
+		},
+		{
+			model: 'htmlTbody',
+			view: 'tbody',
+			modelSchema: {
+				allowIn: 'htmlTable',
+				isBlock: true
+			}
+		},
+		{
+			model: 'htmlThead',
+			view: 'thead',
+			modelSchema: {
+				allowIn: 'htmlTable',
+				isBlock: true
+			}
+		},
+		{
+			model: 'htmlTfoot',
+			view: 'tfoot',
+			modelSchema: {
+				allowIn: 'htmlTable',
+				isBlock: true
+			}
+		},
+		{
+			model: 'htmlCaption',
+			view: 'caption',
+			modelSchema: {
+				allowIn: 'htmlTable',
+				allowChildren: '$text',
+				isBlock: true
+			}
+		},
+		{
+			model: 'htmlTr',
+			view: 'tr',
+			modelSchema: {
+				allowIn: [ 'htmlTable', 'htmlThead', 'htmlTbody' ],
+				isBlock: true
+			}
+		},
+		// TODO can also include text.
```

Comment:
We're keeping the `TODO` comments here, right?

Reply:
I would keep them for now until we cover https://github.com/ckeditor/ckeditor5/pull/9891 so we can review those definitions again and verify how they should behave.